### PR TITLE
Edits suggested by Sergey Yedrikov

### PR DESCRIPTION
**Pull request by mburke5678**
https://docs.google.com/document/d/1Z6L6zVadVH2YcwivAgDg71di__1yo5gUQ4IaGnEu0_k/edit#heading=h.lkc6it25vluk

### CODE REVIEW

**Comment by mburke5678** [comment]
@syedriko This is a first pass at the comments you made on the logging docs. So far, I have implemented smaller changes, typos, and such. 
I will tackle the larger issues in other PR(s).
@QiaolingTang Can you take a look at these changes to make sure I didn't introduce any errors?

File: logging/config/cluster-logging-collector.adoc [hunk 1]
```diff
@@ -5,7 +5,7 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-{product-title} uses Fluentd to collect operations and application logs from your cluster which {product-title} enriches with Kubernetes Pod and Namespace metadata.
+{product-title} uses Fluentd to collect operations and application logs from your cluster  and enriches with Kubernetes Pod and Namespace metadata.
```
Comment (syedriko) [inline logging/config/cluster-logging-collector.adoc]: and enriches them(?) with ...

File: logging/config/cluster-logging-elasticsearch.adoc [hunk 2]
```diff
@@ -7,14 +7,11 @@ toc::[]
 
 {product-title} uses Elasticsearch (ES) to store and organize the log data.
 
-You can configure storage for your Elasticsearch cluster, and 
-define how shards are replicated across data nodes in the cluster, from full replication to no replication.
+some of the modifications you can make to your Elasticsearch deployment include:
```
Comment (syedriko) [inline logging/config/cluster-logging-elasticsearch.adoc]: Capital S?

File: modules/cluster-logging-elasticsearch-tolerations.adoc [hunk 3]
```diff
@@ -54,17 +54,15 @@ that do not match.
     elasticsearch:
       nodeCount: 1
       tolerations: 
-      - key: "1"  <1>
+      - key: "elaseticsearch"  <1>
```
Comment (syedriko) [inline modules/cluster-logging-elasticsearch-tolerations.adoc]: elasticsearch

File: modules/cluster-logging-kibana-tolerations.adoc [hunk 4]
```diff
@@ -5,10 +5,10 @@
 [id="cluster-logging-kibana-tolerations_{context}"]
 = Using tolerations to control the Kibana Pod placement
 
-You can control which nodes the Kibana Pod runs on and prevent 
+You can control which nodes the Kibana Pods runs on and prevent
```
Comment (syedriko) [inline modules/cluster-logging-kibana-tolerations.adoc]: run

File: modules/cluster-logging-log-forwarding-configure.adoc [hunk 5]
```diff
@@ -7,7 +7,7 @@
 
 To configure the Log Forwarding, edit the Cluster Logging Custom Resource (CR) to add the `clusterlogging.openshift.io/logforwardingtechpreview: enabled` annotation and create a Log Forwarding Custom Resource to specify the outputs, pipelines, and enable log forwarding. 
 
-If you enable Log Forwarding, you should define a pipeline all three source types. The logs from any undefined source type are dropped. For example, if you specified a pipeline for the `logs.app` and `log-audit` types, but did not specify a pipeline for the `logs.infra` type, `logs.infra` logs are dropped.
+If you enable Log Forwarding, you should define a pipeline all three source types: `logs.app`, `logs.infra`, and `logs.audit`. The logs from any undefined source type are dropped. For example, if you specified a pipeline for the `logs.app` and `log-audit` types, but did not specify a pipeline for the `logs.infra` type, `logs.infra` logs are dropped.
```
Comment (syedriko) [inline modules/cluster-logging-log-forwarding-configure.adoc]: a pipeline for? all

**Comment by mburke5678** [comment]
@syedriko Thanks. Looks like I misunderstood some of your comments!

File: logging/config/cluster-logging-management.adoc [hunk 6]
```diff
@@ -10,9 +10,9 @@ In order to modify certain components managed by the Cluster Logging Operator or
 
 In unmanaged state, the operators do not respond to changes in the CRs. The administrator assumes full control of individual component configurations and upgrades when in unmanaged state.
 
-In managed state, the Cluster Logging Operator (CLO) responds to changes in the Cluster Logging Custom Resource (CR) and attempts to update the cluster to match the CR.
+In managed state, the Cluster Logging Operator (CLO) responds to changes in the Cluster Logging Custom Resource (CR) and attempts to update the {product-title} cluster to match the CR.
```
Comment (QiaolingTang) [inline logging/config/cluster-logging-management.adoc]: @mburke5678 what does `{product-title}` mean? Is it equal to `OpenShift`? If yes, I think here should be something like `logging cluster` or `clusterlogging`.

File: logging/config/cluster-logging-management.adoc [hunk 7]
```diff
@@ -10,9 +10,9 @@ In order to modify certain components managed by the Cluster Logging Operator or
...
 
-The {product-title} documentation indicates in a prerequisite step when you must set the cluster to Unmanaged.
+The {product-title} documentation indicates in a prerequisite step when you must set the {product-title} cluster to Unmanaged.
```
Comment (QiaolingTang) [inline logging/config/cluster-logging-management.adoc]: Same thing here.

File: logging/config/cluster-logging-management.adoc (hunk 6 above)
Comment (mburke5678) [reply logging/config/cluster-logging-management.adoc]: The {product-title} is a variable that places the OpenShift project in the docs (OpenShift Container Platform, OpenShift Dedicated, OpenShift Online, OKD, and so forth).

Comment (mburke5678) [reply logging/config/cluster-logging-management.adoc]: I changed this to _adjusts the logging deployment accordingly._ to match the wording we use on the **Understanding cluster logging** page. [1]
`
In managed state, the Cluster Logging Operator (CLO) responds to changes in the Cluster Logging Custom Resource (CR) and adjusts the logging deployment accordingly.`

[1] https://docs.openshift.com/container-platform/4.3/logging/cluster-logging.html#cluster-logging-about_cluster-logging

**Pull request merged**